app/javascript/components/Athletes/AthleteRemove.js

```diff
@@ -0,0 +1,43 @@
+import React from "react"
+import PropTypes from "prop-types"
+import gql from 'graphql-tag';
+import { Mutation } from 'react-apollo';
+
+class AthleteRemove extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      mutation: gql`
+        mutation RemoveAthlete($id: ID!) {
+          removeAthlete(id: $id) {
+            id
+          }
+        }
+      `
+    }
+  }
+
+  render() {
+    return (
+      <Mutation mutation={this.state.mutation}>
+        {(removeAthlete, { data }) => (
+          <button onClick={e => {
+            e.preventDefault();
+            
+            removeAthlete({ variables: {
+              id: this.props.athleteId
+            }});
+          }}>
+            Remove
+          </button>
+        )}
+      </Mutation>
+    )
+  }
+}
+
+AthleteRemove.propTypes = {
+  athleteId: PropTypes.string
+};
+
+export default AthleteRemove;
```

Comment:
AddAthlete (as a parent component) should query the data to set the AddAthletes state (`setState()`).
AddAthlete renders AthletesTable where AddAthlete `state.athletes` is passed as a property.
AddAthlete should pass a function to AthleteForm in order to call this function when new athlete is created. (`this.props.handleUpdate(data`) in AthleteForm).
On `handleUpdate` the state of AddAthlete shall be modified, what rerenders the AthleteTable with new params (adds a new row to the table).
Similar thing should happen for RemoveAthlete component (button).

AddAthlete
|- AthletesTable
|- AthleteForm
'-  RemoveAthlete